src/components/tweet.js

```diff
@@ -62,6 +64,92 @@ export default class Tweet extends React.Component {
     });
   }
 
+  openMediaInWindow (med) {
+    let image = new Image();
+    let loaded = false;
+    var mediaUrl = 0;
+    var media = med.video_info;
+    if (med.video_info != null) {
+      mediaUrl = med.video_info.variants[0].url;
+      var mediaType = med.video_info.variants[0].content_type;
+    } else {
+      mediaUrl = med.media_url;
+    }
+    image = document.getElementById('tweet_mediaid');
+    image.onload = function () { loaded = true; };
+    let win = new BrowserWindow({titleBarStyle: 'hidden', x:(screen.width/2)-(450/2),y:(screen.height/2)-(450/2), resizable: false, width: 100, height: 100});
+    win.loadURL('file://' + __dirname + '../..' + '/mediaPopup.html');
+    if (loaded) {
+      clearInterval(wait);
+    } else {
+      win.webContents.executeJavaScript(`
+        var ipcRenderer = require('electron').ipcRenderer;
+        var img = document.getElementById('loadedImage');
+        var video = document.getElementById('loadedVideo');
+        var mediaWidth = 0,
+        mediaHeight = 0,
+        maxWidth = 800,
+        maxHeight = 600;
+
+        function resizeMedia(media){
```

Comment:
The function does only what it's supposed to, i'm not sure what you'll gain from splitting it into two functions. 

First it looks for the media, if it's a img, put this.width into our var mediaWidth and same with mediaHeight, else put this.videoHeight into mediaWidth, then we start looking if our media input is larger than the maxWidth we have specified, if it is, scale it down.

The last if statement checks if our img width is more than 0, if so set this.width (the image width, which is the same as saying img.width = mediaWidth, we just refer to the input of the function.